data/spells/scripts/support/levitate.lua

```diff
@@ -0,0 +1,36 @@
+function onCastSpell(creature, variant)
+	local currentPosition = creature:getPosition()
+	local destinationPosition = creature:getPosition()
+	destinationPosition:getNextPosition(creature:getDirection())
+
+	local returnValue = RETURNVALUE_NOTPOSSIBLE
+	local parameter = variant:getString():lower()
```

Comment:
I don't there is a need to keep consistency between talkactions and spells, in any case, it would be even better if we renamed talkaction's parameter ``param`` to ``parameter``.